main.js

```diff
@@ -1,46 +1,48 @@
-const core = require('@actions/core')
-const yaml = require('js-yaml')
-const fs = require('fs')
+const core = require("@actions/core");
+const yaml = require("js-yaml");
+const fs = require("fs");
 
-async function run () {
+async function run() {
   try {
-    const files = core.getInput('files').split(',')
+    const files = core.getInput("files").split(",");
     //   TODO: verify the proper files were passed and learner didn't tamper with grading.yml
 
     const answers = {
-      'stale-daily': '0 0 * * *',
-      'stale-weekly': '0 0 * * MON',
-      'stale-monthly': '0 0 1 * *'
-    }
+      "stale-daily": "0 0 * * *",
+      "stale-weekly": ["0 0 * * MON", "0 0 * * 1"],
+      "stale-monthly": "0 0 1 * *",
+    };
 
     files.forEach((file) => {
-      const filename = file.replace('.yml', '')
+      const filename = file.replace(".yml", "");
 
       const doc = yaml.load(
         fs.readFileSync(
-        `${process.env.GITHUB_WORKSPACE}/.github/workflows/${file}`,
-        'utf8'
+          `${process.env.GITHUB_WORKSPACE}/.github/workflows/${file}`,
+          "utf8"
         )
-      )
+      );
+
       // TODO: if desired keys dont' exist prevent failure but provide feedback
-      if (answers[filename].trim() === doc.on.schedule[0].cron.trim()) {
-        console.log(`Results for ${filename}: correct`)
+      if (answers[filename].trim().includes(doc.on.schedule[0].cron.trim())) {
```

Comment:
@mattdavis0351 I believe this line is throwing an error ([source](https://github.com/hectorsector/improved-octo-tribble/runs/2004227048?check_suite_focus=true#step:3:7)):
```
Error: TypeError: answers[filename].trim is not a function
```

Reply:
Doh!  I've made an update 🤦 .  This is why we need testing